src/status_im/subs/wallet/networks.cljs

```diff
@@ -63,6 +63,17 @@
                   :layer            layer)))
         (sort-by (juxt :layer :short-name)))))
 
+(re-frame/reg-sub
+ :wallet/network-details-by-network-name
+ :<- [:wallet/network-details]
+ (fn [network-details]
+   (when (seq network-details)
+     (->> network-details
+          (group-by :network-name)
+          (map (fn [[network-key network-group]]
+                 [network-key (first network-group)]))
+          (into {})))))
+
```

Comment:
Here we're creating a map of the network details which can be accessed by the network name.
TODO: add tests 🚧